Add pickADate and pickATime providers

jayellsbrg opened the pull request:
For setting default options, like format, for an entire module.

Judging from the open issues, you're working on this in your next major version, but I thought this feature would be useful now since it shouldn't detract from existing functionality.

jayellsbrg committed:
Add pickADate and pickATime providers

For setting default options, like format, for an entire module.

Toilal commented:
Great ! Could you add a commit with README.md update to show an example of how it works ? It would fit in the Options section, just after the existing.

jayellsbrg committed:
Update README.md

Add instructions for how to use new pickADateProvider and pickATimeProvider to set default options for all date pickers.

jayellsbrg commented:
I intended to, but had to rush out the door earlier. :) README.md updated.
